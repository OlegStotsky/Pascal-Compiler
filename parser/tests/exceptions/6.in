var
a : integer;
b : integer;

begin
	for i := 1 to a<b do
		a := a +1;
end;