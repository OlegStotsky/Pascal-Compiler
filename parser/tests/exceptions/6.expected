Error at line 5, column 21 : integer expected but boolean found
